float32 x
float32 y
float32 z
bool hand_detected
bool is_peace
bool is_dino
bool is_open
